examples/config-many-files/teams.yml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: teams
+spec:
+  teams:
+  - name: workstations
+    hosts:
+      - hardware_serial1
+      - hwardware_serial2
+      - hwardware_serial3
+    users:
+      - email1
```

Comment:
Same comment as about `hosts` above.

Also - I imagine this is sort of a v0, but I'm starting to think about what definitions of roles for RBAC will look like.

Reply:
Good point. I've added the `role` key in commit 63fca2b. You're right in that this is sort of a v0. The `teams.yml` config file closely aligns with the draft `/teams` API endpoint described in the [REST API for multiple teams draft PR](https://github.com/fleetdm/fleet/pull/383/files#diff-c905df2abfb396957c083e6a513b65e83dca6f4acfedf6320749a3002efd16ffR4172).